tests/scripts/thread-cert/node.py

```diff
@@ -1765,10 +1765,14 @@ def ping(self, ipaddr, num_responses=1, size=None, timeout=5):
         result = True
         # ncp-sim doesn't print Done
         done = (self.node_type == 'ncp-sim')
-        while len(responders) < num_responses or not done:
+
+        # ncp-sim doesn't print statistics
+        received_statistics = (self.node_type == 'ncp-sim')
+        is_multicast = ipaddr.startswith('ff')
```

Comment:
```suggestion
        is_multicast = ipaddress.IPv6Address(ipaddr).is_multicast
```
nit: use [ipaddress library](https://docs.python.org/3/library/ipaddress.html)